src/Parsing/String.purs

```diff
@@ -195,6 +195,9 @@ match p = do
 
 -- | Compile a regular expression `String` into a regular expression parser.
 -- |
```

Comment:
This is good, thanks, but I'd rather have it down below in the **Flags** section than right up at the top of the docs.